oeps/oep-0004.rst

```diff
@@ -121,6 +121,21 @@ Governance
 * Resources should not, on the other hand, be too broad to be useful, i.e.,
   instead of ``courseware``, consider ``discussions`` or ``catalog`` or
   whatever specific resource is of interest
+ 
+* Scope descriptions need to be presented to the user by the token management
+  system.  They should therefore be maintained with the central list of scopes.
+  Scope descriptions should be internationalized, but should be presented with
+  the scope name, which is consistent across translations.
+
+Alt:
+
+* Scope metadata should be maintained centrally with the canonical list of 
+  scopes.
+
+* Scope descriptions should be internationalized strings, managed by Transifex.
+
+* Scopes should be presented to the user during authorization as a combination 
+  of the canonical name and the internationalized description.
```

Comment:
@jcdyer are you planning on finishing this PR or should I have somebody else pick it up?